bpf/flows.c

```diff
@@ -107,28 +108,13 @@ static inline int flow_monitor(struct __sk_buff *skb, u8 direction) {
         aggregate_flow->dns_record.flags = pkt.dns_flags;
         aggregate_flow->dns_record.latency = pkt.dns_latency;
         aggregate_flow->dns_record.errno = dns_errno;
-        long ret = bpf_map_update_elem(&aggregated_flows, &id, aggregate_flow, BPF_ANY);
```

Comment:
this part (removing unnecessary bpf_map_update_elem) is left incomplete on purpose, I'll wait that @msherif1234 's PR is merged then rebase this one